console/src/App/AppContextProvider.js

```diff
@@ -104,7 +104,7 @@ class AppContextProvider extends Component {
 		};
 	}
 
-	componentWillMount () {
+	UNSAFE_componentWillMount () { // https://reactjs.org/docs/react-component.html#unsafe_componentwillmount
```

Comment:
Maybe we should replace all these unsafe methods with safe ones. But I will let the next reviewer give an opinion also